torch/_inductor/cpu_vec_isa.py

```diff
@@ -338,7 +356,10 @@ def valid_vec_isa_list() -> List[VecISA]:
     elif arch == "ppc64le":
         isa_list.append(VecVSX())
     elif arch == "aarch64":
-        isa_list.append(VecNEON())
+        if torch.cpu._is_arm_sve_supported():
+            isa_list.append(VecSVE())
```

Comment:
Is this check sufficient? Do we need to add the check like `cpuinfo_get_max_arm_sve_length == 256` ?https://github.com/pytorch/pytorch/pull/119571/files#diff-54c373491da67eb31c3777457d7b043a49dd3966412edfd928ffd2013e4d6a54R39-R47

Reply:
@CaoE @aditew01 Thanks for your suggestions, I agree. I will include this check un my next SVE PR (Add support to SVE 128/SVE 512). I had to implement the API from here: aten/src/ATen/cpu/Utils.cpp

Here's the snapshot of the logic: (I have verified it's working)
![image](https://github.com/user-attachments/assets/c57dc6b4-f2bb-4839-a2d1-e660828d15e2)